src/it/scala/io/iohk/ethereum/sync/FastSyncItSpec.scala

```diff
@@ -87,394 +48,16 @@ class FastSyncItSpec extends FlatSpecBase with Matchers with BeforeAndAfter {
-    ): (Block, BigInt, InMemoryWorldStateProxy) = {
-      val newBlockNumber = parent.header.number + 1
-      val newWorld = updateWorldForBlock(newBlockNumber, parentWorld)
-      val newBlock = parent.copy(header =
-        parent.header.copy(parentHash = parent.header.hash, number = newBlockNumber, stateRoot = newWorld.stateRootHash)
-      )
-      val newTd = newBlock.header.difficulty + parentTd
-      (newBlock, newTd, parentWorld)
-    }
-
-    def importBlocksUntil(
-        n: BigInt
-    )(updateWorldForBlock: (BigInt, InMemoryWorldStateProxy) => InMemoryWorldStateProxy): Task[Unit] = {
-      Task(bl.getBestBlock()).flatMap { block =>
-        if (block.number >= n) {
-          Task(())
-        } else {
-          Task {
-            val currentTd = bl.getTotalDifficultyByHash(block.hash).get
-            val currentWolrd = getMptForBlock(block)
-            val (newBlock, newTd, newWorld) = createChildBlock(block, currentTd, currentWolrd)(updateWorldForBlock)
-            bl.save(newBlock, Seq(), newTd, saveAsBestBlock = true)
-            bl.persistCachedNodes()
-            broadcastBlock(newBlock, newTd)
-          }.flatMap(_ => importBlocksUntil(n)(updateWorldForBlock))
-        }
-      }
-    }
-
-    def startFastSync(): Task[Unit] = Task {
-      fastSync ! FastSync.Start
-    }
-
-    def waitForFastSyncFinish(): Task[Boolean] = {
-      retryUntilWithDelay(Task(storagesInstance.storages.appStateStorage.isFastSyncDone()), 1.second, 90) { isDone =>
-        isDone
-      }
-    }
-
-    // Reads whole trie into memory, if the trie lacks nodes in storage it will be None
-    def getBestBlockTrie(): Option[MptNode] = {
-      Try {
-        val bestBlock = bl.getBestBlock()
-        val bestStateRoot = bestBlock.header.stateRoot
-        MptTraversals.parseTrieIntoMemory(
-          HashNode(bestStateRoot.toArray),
-          storagesInstance.storages.stateStorage.getBackingStorage(bestBlock.number)
-        )
-      }.toOption
-    }
-  }
-
-  object FakePeer {
-    def startFakePeer(peerName: String): Task[FakePeer] = {
-      for {
-        peer <- Task(new FakePeer(peerName))
-        _ <- peer.startPeer()
-      } yield peer
-    }
-
-    def start1FakePeerRes(): Resource[Task, FakePeer] = {
-      Resource.make {
-        startFakePeer("Peer1")
-      } { peer =>
-        peer.shutdown()
-      }
-    }
-
-    def start2FakePeersRes() = {
-      Resource.make {
-        Task.parZip2(startFakePeer("Peer1"), startFakePeer("Peer2"))
-      } { case (peer, peer1) => Task.parMap2(peer.shutdown(), peer1.shutdown())((_, _) => ()) }
-    }
-
-    def start3FakePeersRes() = {
-      Resource.make {
-        Task.parZip3(startFakePeer("Peer1"), startFakePeer("Peer2"), startFakePeer("Peer3"))
-      } { case (peer, peer1, peer2) =>
-        Task.parMap3(peer.shutdown(), peer1.shutdown(), peer2.shutdown())((_, _, _) => ())
-      }
-    }
-  }
-}
+}
```

Comment:
` sbt test:scalastyle` complains about lack of newline at the end here and couple of other places:

```shell
[error] /mantis/src/it/scala/io/iohk/ethereum/sync/util/FakePeer.scala: File must end with newline character
[error] /mantis/src/it/scala/io/iohk/ethereum/sync/util/FakePeerFastSync.scala: File must end with newline character
[error] /mantis/src/it/scala/io/iohk/ethereum/sync/util/FakePeerRegularSync.scala: File must end with newline character
[error] /mantis/src/it/scala/io/iohk/ethereum/sync/FastSyncItSpec.scala: File must end with newline character
[error] /mantis/src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala: File must end with newline character
```